docs/debug-kernel.md

```diff
@@ -0,0 +1,33 @@
+# Get Clear Containers Kernel logs
+
+Clear Containers runtime launch a special virtual machine when a new pod is
+created. This VM uses a custom kernel inside the VM.
+
+The kernel boot logs are disabled by default. However when these logs may be
+desired when the kernel is customized by the end user, or if the virtual machine
+boot fails due to the kernel.
+
+To provide a debug log of the kernel boot, Clear Containers supports an option
+to enable kernel boot logs using the configuration.
+
+1- Enable debug in configuration.toml
+
+```
+enable_debug = true
+```
+
+2- Run a container to generate the logs
+
+```
+sudo docker run -ti busybox true
+```
+
+3- Filter the kernel boot logs from the `cc-proxy` logs
+The `cc-proxy` logs show the sources of its collated information. To see
+the kernel boot logs, filter `cc-proxy` logs by the QEMU serial console.
+The QEMU serial console is represented by `source=qemu`.
+
+```
+journalctl -u cc-proxy | grep source=qemu
+```
+
```

Comment:
did you also mention you could hand-run the proxy and get less verbose/encoded logs out of it? iirc, the kernel logs in the journalctl proxy log are rather hard to trawl through and read.

Reply:
Added that detail.